src/vtbackend/Capabilities_test.cpp

```diff
@@ -27,7 +27,7 @@ TEST_CASE("Capabilities.get")
     REQUIRE(rgb == "8/8/8");
 
     auto const colors = tcap.numericCapability("colors");
-    REQUIRE(colors == 256);
+    REQUIRE(colors == 0x7FFF);
```

Comment:
can we use here numerical limit, i am very bad at hex numbers :)

Reply:
done